Tasks/cURLUploader/task.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "AD8974D8-DE11-11E4-B2FE-7FB898A745F3",
+    "name": "cURLUploader",
+    "friendlyName": "cURL Upload Files",
+    "description": "Use cURL to upload files with supported protocols. (FTP, FTPS, SFTP, HTTP, and more)",
+    "category": "Utility",
```

Comment:
In my opinion the category should be "Deploy" since that already contains tasks like "Copy Files to Machines" and "Azure Web Site Deployment".  The "Utility" category does contain a task called "Copy Files to Azure Blob", but these tasks seem more related to deployment than utility to me.  Thoughts?